### PR TITLE
Provide documentation for the on-release GH action

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by dagda1** [comment]
@mareklibra Please do not take this the wrong way but I think you could save yourself a whole pile of development by just using [changesets](https://github.com/changesets/changesets) which backstage uses or if the eventual home of the plugin is [janus-idp](https://github.com/janus-idp/backstage-plugins), it uses something called [semantic-release](https://github.com/semantic-release/github) which is something I've not heard of but looks interesting.

Every line of code we create needs maintained and will need updated as requirements change, e.g. we have another plugin (I'm working on another one now for the auth).  

Changesets and semantic-release are battle tested and will require no extra dev.

**Comment by dagda1** [comment]
changesets will update all the versions in all the package.json files, e.g. the package.json in `package/app/package.json` will need updated to the new `@parodos/parodos-plugin` version.

Personally, I don't want to write code like this and would rather use a package that does this.

File: plugins/parodos/README.md [hunk 1]
```diff
@@ -61,3 +61,24 @@ proxy:
...
+To do a release:
+
+- go to the project release page: https://github.com/parodos-dev/backstage-parodos/releases
+- click **Create a new release**
+- as a tag, use format **vX.Y.Z** (mind the **v** prefix!)
+- as a title, use the same value as for the tag
+- let the release notes to be generated, adjust
+- click **Publish release**
+
+The on-release GitHub action is trigerred, leading to publishing the new version to the NPM registry.
```
Comment (masayag) [inline plugins/parodos/README.md:78]: s/trigerred/triggered

File: plugins/parodos/README.md [hunk 2]
```diff
@@ -61,3 +61,24 @@ proxy:
...
+
+To watch progress: https://github.com/parodos-dev/backstage-parodos/actions/workflows/on-release.yaml
+
+If the action is failing on expired token, they can be updated here: https://github.com/organizations/parodos-dev/settings/secrets/actions
```
Comment (masayag) [inline plugins/parodos/README.md:82]: s/they/it

**Comment by dagda1** [comment]
What do you do when there are 2 plugins?  More code?

**Comment by dagda1** [comment]
@mareklibra at the end of the day it is really up to you guys what you do but I have created a new plugin for the auth because the auth should be separate so consumers of the main plugin have the option to use their own and there is also separation of concerns etc.

For me simple is writing no code and as I said, at the end of the day, it is really down to you.

**Comment by mareklibra** [comment]
The description conforms recent procedure, so merging.
We will evolve it based on changing needs.

**Pull request merged**